javascript/src/router.js

```diff
@@ -36,6 +56,21 @@ function routeAppliesToCurrentLocation(route) {
   return r.match(page.current, {});
 }
 
+/**
+ * Find base url if available
+ *
+ * @returns {string}
+ */
+function getAbsoluteBase() {
+  const baseTags = window.document.getElementsByTagName('base');
+  if (baseTags && baseTags[0]) {
+    return baseTags[0].href;
+  }
+  return null;
+}
+
+page.getAbsoluteBase = getAbsoluteBase.bind(page);
+page.normalise = normalise.bind(page);
```

Comment:
I think we should support it, since it's going to be a standard route mapping between "traditional SS urls" and "paths expected by page.js". It will probably live as long as page.js does...